.gitignore

```diff
@@ -20,3 +20,4 @@ experimental
 notes.txt
 build
 *venv
+.idea/*
```

Comment:
add new line at the end of the file (you may want to configure your text editor so that it's done automatically?)